qutip/bloch3d.py

```diff
@@ -262,33 +267,39 @@ def add_states(self, state, kind='vector'):
         kind : str {'vector','point'}
             Type of object to plot.
 
+        alpha : float, default=1.
+            Transparency value for the vectors. Values between 0 and 1.
         """
         if isinstance(state, Qobj):
             state = [state]
         for st in state:
             if kind == 'vector':
                 vec = [expect(sigmax(), st), expect(sigmay(), st),
                        expect(sigmaz(), st)]
-                self.add_vectors(vec)
+                self.add_vectors(vec, alpha=alpha)
             elif kind == 'point':
                 pnt = [expect(sigmax(), st), expect(sigmay(), st),
                        expect(sigmaz(), st)]
-                self.add_points(pnt)
+                self.add_points(pnt, alpha=alpha)
 
-    def add_vectors(self, vectors):
+    def add_vectors(self, vectors, alpha=1.0):
         """Add a list of vectors to Bloch sphere.
 
         Parameters
         ----------
         vectors : array/list
             Array with vectors of unit length or smaller.
 
+        alpha : float, default=1.
+            Transparency value for the vectors. Values between 0 and 1.
+
         """
         if isinstance(vectors[0], (list, np.ndarray)):
             for vec in vectors:
                 self.vectors.append(vec)
         else:
             self.vectors.append(vectors)
+        self.vector_alpha.append(alpha)
```

Comment:
Similar critique here to `add_vectors` for `Bloch`. We need to add an alpha value for each vector.